packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard.lua

```diff
@@ -270,38 +155,31 @@ function clean_lime_config()
     f:close()
 end
 
+-- Apply configuraation permanenty
+-- TODO: check if config is valid
+-- TODO: use safe-reboot
 function apply_config(file)
-    -- TODO: check if config is valid
     local filePath = "/tmp/"..file
-    check_file_exists(filePath)
-    -- nixio.syslog("crit", "FBW FILE EXISTS ")
-    execute("rm /etc/config/lime")
-    execute("cp "..filePath.." /etc/config/lime-defaults")
-    -- nixio.syslog("crit", "FBW FILE COPIED ")
+    check_utils.file_exists(filePath)
+    -- Clean previus lime configuration and replace lime-defaults
     clean_lime_config()
-    -- nixio.syslog("crit", "FBW LIME CONFIG CLEANED ")
-    execute("/rom/etc/uci-defaults/91_lime-config")
-    execute("rm /etc/first_run")
-    -- nixio.syslog("crit", "APPLY CONFIGS ")
+    utils.execute("cp "..filePath.." /etc/config/lime-defaults")    
+    -- Run lime-config as first boot
+    utils.execute("/rom/etc/uci-defaults/91_lime-config")
+    -- Remove FBW lock file
+    utils.execute("rm /etc/first_run")
+    -- Apply new configuration
     os.execute("(( /usr/bin/lime-config && /usr/bin/lime-apply && reboot 0<&- &>/dev/null &) &)")
 end
 
-function filter_mesh(n)
-    return n.mode == "Ad-Hoc" or n.mode == "Mesh Point"
-end
-
-local function start_scan_file()
-    local file = io.open("/tmp/scanning", "w")
-    file:write("true")
-    file:close()
-end
-
-local function stop_scan()
+-- Remove scan lock file
+local function end_scan()
     local file = io.open("/tmp/scanning", "w")
     file:write("false")
     file:close()
 end
 
+-- Read scan status
 function check_scan_file()
```

Comment:
this function is not used, so should be removed

Reply:
Its not used here, but is used by ubus deamon